src/scenes/load.js

```diff
@@ -90,7 +90,7 @@ export class LoadScene extends Phaser.Scene {
         this.load.image("red-hitsplat", "src/assets/effects/RedHitsplat.png");
 
         // Enemies
-        this.load.image("chicken", "src/assets/sprites/Chicken.jpg");
+        this.load.image("chicken", "src/assets/sprites/Chicken.png");
```

Comment:
Probably wasn't before since it was a jpg but since we're converting it maybe get a transparent one if you can find one